src/raftstore/store/fsm/batch.rs

```diff
@@ -214,8 +226,27 @@ impl<N: Fsm, C: Fsm> Batch<N, C> {
             control_box.release(s);
         }
     }
-}
 
+    pub fn check_batch(&mut self, check_size: &mut usize, scheduler: NormalScheduler<N, C>) {
+        let mut v: Vec<usize> = Vec::new();
+        for (i, count) in self.count.iter_mut().enumerate() {
+            if count > check_size {
+                v.push(i);
+            }
+            *count += 1;
+        }
+        while let Some(index) = v.pop() {
```

Comment:
Is `v` sorted? Otherwise it could be wrong.

Reply:
the data of v is ordered, from small to large. We first remove the back fsm, it will not affect the index of other fsm,similar stack.